monitoring/opencensus/requirements.txt

```diff
@@ -0,0 +1,13 @@
+flake8==3.8.4
+Flask==1.1.2
+google-api-core==1.25.1
+google-auth==1.25.0
+googleapis-common-protos==1.52.0
+nox==2020.12.31
+opencensus==0.7.12
+opencensus-context==0.1.2
+opencensus-ext-prometheus==0.2.1
+prometheus-client==0.9.0
+prometheus-flask-exporter==0.18.1
+pytest==6.2.2
+requests==2.25.1
```

Comment:
I think there are a few more that could be deleted:
* `flake8` - only needed for linting
* `nox` - the test runner
* `pytest` - for testing

Reply:
Done, thank you.

Reply:
RE: region tags - I may need to add those once the Cloud Docs folks start to incorporate this into the doc, but thank you for pointing that out.